iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/MergedBus.java

```diff
@@ -81,6 +81,17 @@ public int getConnectedTerminalCount() {
         return buses.stream().flatMap(ConfiguredBus::getConnectedTerminalStream);
     }
 
+    @Override
+    public TerminalExt getTerminalReference() {
+        checkValidity();
+        return getConnectedTerminalStream()
+                .findFirst()
+                .orElseGet(() -> buses.stream()
+                        .map(BusExt::getTerminalReference)
+                        .findFirst()
+                        .orElse(null));
```

Comment:
same question here... (if there's no connected terminal, then the terminal reference of the buses are disconnected terminals)

Reply:
Yes, in this case you are right, `MergedBus` are buses in the `BusView` so either the terminal reference is a connected terminal either there is none since there is no switch in this view